clusters/build-clusters/01_cluster/assets/prom_prow_rules.yaml

```diff
@@ -0,0 +1,147 @@
+          on (namespace, pod)
+          group_right (node, pod_ip, phase)
+          max by (namespace, pod) (kube_pod_created)
+    # time since a pod has been in pending phase
+    # {namespace="...",pod="...",node="...",pod_ip="1.2.3.4",phase="Pending"} [unix timestamp]
+    - record: prow:pod:pending_since_time
+      expr: prow:pod:pending * prow:pod:created_time
+
+    # These metrics are specific to Prow jobs.
+
+    # This monster of a rule prepares a tidy list of jobs with their
+    # namespace and pod name attached.
+    # {namespace="...",pod="...",node="...",pod_ip="1.2.3.4",phase="...",org="...",repo="...",type="...",name="...",id="<UUID>"} 1
+    - record: prow:job
+      expr: |
+        max by (org, repo, type, name, id, namespace, pod, phase, node, pod_ip) (
+          label_replace(
+            label_replace(
+              label_replace(
+                label_replace(
+                  label_replace(
+                    prow:pod *
+                      on (namespace, pod)
+                      group_left (label_prow_k8s_io_id, label_prow_k8s_io_refs_repo, label_prow_k8s_io_type, label_prow_k8s_io_job, label_prow_k8s_io_refs_org)
+                      kube_pod_labels{label_prow_k8s_io_id!=""},
+                    "id",
+                    "$0",
+                    "label_prow_k8s_io_id",
+                    ".*"
+                  ),
+                  "repo",
+                  "$0",
+                  "label_prow_k8s_io_refs_repo",
+                  ".*"
+                ),
+                "type",
+                "$0",
+                "label_prow_k8s_io_type",
+                ".*"
+              ),
+              "name",
+              "$0",
+              "label_prow_k8s_io_job",
+              ".*"
+            ),
+            "org",
+            "$0",
+            "label_prow_k8s_io_refs_org",
+            ".*"
+          )
+        )
+    # list of all pending pods
+    # {namespace="...",pod="...",node="...",pod_ip="1.2.3.4",phase="Pending",org="...",repo="...",type="...",name="...",id="<UUID>"} 1
+    - record: prow:job:pending
+      expr: |
+        prow:pod:pending *
+          on (namespace, pod)
+          group_left (org, repo, type, name, id)
+          prow:job
+    # time since a job has been in pending phase
+    # {namespace="...",pod="...",node="...",pod_ip="1.2.3.4",phase="Pending",org="...",repo="...",type="...",name="...",id="<UUID>"} [unix timestamp]
+    - record: prow:job:pending_since_time
+      expr: |
+        prow:pod:created_time *
+          on (namespace, pod)
+          group_left (org, repo, type, name, id)
+          prow:job:pending
+    # total memory used by a Prow job
+    # {namespace="...",pod="...",node="...",pod_ip="1.2.3.4",phase="...",org="...",repo="...",type="...",name="...",id="<UUID>"} [bytes]
+    - record: prow:job:memory_working_set_bytes
+      expr: |
+        sum by (namespace, pod, node, pod_ip) (container_memory_working_set_bytes{container="test"}) *
```

Comment:
In the past when we've been bitten by usage we got screwed on `sidecar` so why filter to only one container in the Pod?

Reply:
Also. max RSS is a separate metric and sensitive to spiky behavior. whereas this is not

Reply:
added the container to the possible targets